### PR TITLE
remove platinum-sw

### PR DESCRIPTION
**Pull request by e111077**
Needed for PolymerElements/app-pouchdb#50

### CODE REVIEW

File: service-worker.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// START OF AUTO-GENERATED SW-TOOLBOX BY SW-PRECACHE CLI
+
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (valdrinkoshi) [inline service-worker.js:4]: Is this expected to be 2016? Does it update to 2017 if you re-generate it again?

File: bower.json [hunk 2]
```diff
@@ -8,7 +8,6 @@
   "license": "http://polymer.github.io/LICENSE.txt",
   "private": true,
   "dependencies": {
-    "platinum-sw": "polymerelements/platinum-sw#1 - 2",
```
Comment (valdrinkoshi) [inline bower.json:11]: Normally they should be declaring this as their dependency too, but some users might anyways rely on this transitive dependency. We should at least do a minor bump to the version once this gets merged (aka to 2.1.0)

File: service-worker.js (hunk 1 above)
Comment (e111077) [reply service-worker.js:4]: It doesn't seem to update to 2017

File: bower.json (hunk 2 above)
Comment (e111077) [reply bower.json:11]: sounds good

File: bower.json [hunk 3]
```diff
@@ -19,13 +18,13 @@
     "paper-fab": "polymerelements/paper-fab#1 - 2",
     "paper-icon-button": "polymerelements/paper-icon-button#1 - 2",
     "iron-input": "polymerelements/iron-input#1 - 2",
-    "iron-autogrow-textarea": "polymerelements/iron-autogrow-textarea#1 - 2"
+    "iron-autogrow-textarea": "polymerelements/iron-autogrow-textarea#1 - 2",
+    "web-animations-js": "web-animations/web-animations-js#^2.2.0"
```
Comment (valdrinkoshi) [inline bower.json]: Where is `web-animations-js` needed?

Comment (e111077) [reply bower.json]: `na-editor.html` uses `element.animate`. I never actually put the HTML import in there. ~I should do that...~

**Comment by FluorescentHallucinogen** [comment]
The [codelab](https://codelabs.developers.google.com/codelabs/polymer-firebase-pwa/) must be updated too, because it mentions `platinum-sw`. Can anyone please update it?

BTW, how to get the access to update codelabs? :thinking:

**Comment by FluorescentHallucinogen** [comment]
Any progress on it?

**Comment by e111077** [comment]
I sent a request to pin the bower installs to 1.x

File: bower.json (hunk 3 above)
Comment (e111077) [reply bower.json]: Actually I don't include the import. this just makes it available for the codelab for convenience

See:
https://codelabs.developers.google.com/codelabs/polymer-firebase-pwa/#9

Comment (valdrinkoshi) [reply bower.json]: oh i see, LGTM 👍

**Pull request merged**